### PR TITLE
feat: improve service state 

### PR DESCRIPTION
**Pull request by mohandast52**
three

Screenshot 2023-02-27 at 4 10 40 PM

### CODE REVIEW

File: common-util/Details/ServiceState/helpers.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const SERVICE_STATE_HELPER_LABELS = {
+  'pre-registration': 'The service has just been registered.',
```
Comment (jmoreira-valory) [inline common-util/Details/ServiceState/helpers.jsx:2]: *minted

File: common-util/Details/ServiceState/helpers.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  'active-registration':
+    'The service is waiting for agent operators to register their agent instances on the service.',
```
Comment (jmoreira-valory) [inline common-util/Details/ServiceState/helpers.jsx:4]: 'The service is waiting for operators to register their agent instances.'

Comment (jmoreira-valory) [inline common-util/Details/ServiceState/helpers.jsx:4]:
```suggestion
    'The service is waiting for agent operators to register their agent instances.',
```

File: common-util/Details/ServiceState/helpers.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  'finished-registration':
+    'All agent instance slots have been filled and waiting for the service owner to continue deploying the service.',
```
Comment (jmoreira-valory) [inline common-util/Details/ServiceState/helpers.jsx:6]:
```suggestion
    'All agent instance slots have been filled. Waiting for the service owner to continue deploying the service.',
```

File: common-util/Details/ServiceState/helpers.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  deployed:
+    'The service is in default operational state and the agent operators can turn on their agent instances at this point.',
```
Comment (jmoreira-valory) [inline common-util/Details/ServiceState/helpers.jsx:8]:
```suggestion
    'The service is in default operational state. Agent operators can turn on their agent instances at this point.',
```

File: common-util/Details/ServiceState/helpers.jsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  terminated:
+    'The service has been terminated by the service owner and the service is waiting for the owner to unbond all registered agents.',
```
Comment (jmoreira-valory) [inline common-util/Details/ServiceState/helpers.jsx:10]:
```suggestion
    'The service has been terminated by the service owner. Waiting for the service owner to unbond all registered agents.',
```

File: common-util/Details/ServiceState/helpers.jsx (hunk 1 above)
Comment (jmoreira-valory) [inline common-util/Details/ServiceState/helpers.jsx:2]:
```suggestion
  'pre-registration': 'The service has just been minted.',
```

**Review by jmoreira-valory — APPROVED** [review]
LGTM. Just check with @oaksprout if he is happy with displaying the info this way.

**Review by oaksprout — APPROVED** [review]

**Pull request merged**